src/main/java/com/yetanalytics/xapi/model/Account.java

```diff
@@ -3,6 +3,10 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 
+/**
+* Class representation of the Account Component of the 
+* <a href="https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#2424-account-object">9274.1.1 xAPI Specification</a>.
+*/
```

Comment:
Is there a way to actually read the Javadoc right now? I'd rather read that and click on links rather than read docstrings in code and having to manually copy-paste URLs.

Reply:
Ah the github-pages publishes the Javadocs.

Reply:
yes. see slack